package/kedro_viz/integrations/deployment/s3_deployer.py

```diff
@@ -0,0 +1,69 @@
+"""`kedro_viz.integrations.deployment.s3_deployer` defines
+deployment class for S3"""
+
+import json
+import logging
+from datetime import datetime
+from pathlib import Path
+
+import fsspec
+from kedro.io.core import get_protocol_and_path
+from semver import VersionInfo
```

Comment:
I see earlier you are using `packaging`, I think we should be consistent using either that of `semver`. And if I remember correctly I read it in some other issue that `packaging` is a better option?

Reply:
Great point. I have reopened the issue and we should replace it. https://github.com/kedro-org/kedro-viz/issues/1460